shared/chat/conversation/messages/set-explode-popup/index.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+import * as React from 'react'
+import {messageExplodeDescriptions} from '../../../../constants/chat2'
+import {type MessageExplodeDescription} from '../../../../constants/types/chat2'
+import {Box, Icon, Text, FloatingMenu} from '../../../../common-adapters'
+import {collapseStyles, globalColors, globalMargins, globalStyles} from '../../../../styles'
+
+const sortedDescriptions = messageExplodeDescriptions.sort((a, b) => (a.seconds < b.seconds ? 1 : 0))
+
+type HeaderProps = {
+  isNew: boolean,
+}
+
+const Header = (props: HeaderProps) => (
+  <Box style={headerContainerStyle}>
```

Comment:
lets start using Box2 and getting rid of these inner margins and use gap instead